src/settings.js

```diff
@@ -518,6 +518,8 @@ const SCHEMA = {
     loginProvided: 'boolean',
   },
   upload: {
+    enableSeries: 'string',
+    enableDescription: 'string',
```

Comment:
Those should be `boolean`s and not strings. Also, the name is a bit misleading, maybe rather something like `userProvidedDescription` or `letUserSelectSeries`.